get_all_manifests.sh

```diff
@@ -19,6 +19,7 @@ declare -A COMPONENT_MANIFESTS=(
     ["odh-model-controller"]="opendatahub-io:odh-model-controller:release-0.12.0:config:odh-model-controller"
     ["kserve"]="opendatahub-io:kserve:release-v0.11.1:config:kserve"
     ["modelregistry"]="opendatahub-io:model-registry-operator:main:config:model-registry-operator"
+    ["trainingoperator"]="z103cb:training-operator:rhoai-manifests:manifests:trainingoperator"
```

Comment:
we should wait manifest PR gets into `opendathub-io` first  to dev branch and remove `z103cb`+` rhoai-manifests`

Reply:
yes. let's make this line the last thing to update.

Reply:
i've updated this in https://github.com/opendatahub-io/opendatahub-operator/pull/944/commits/baebfc6c6255151c8655fad82c9cd4074bb8bfdc  after https://github.com/opendatahub-io/training-operator/pull/3 is merged